### PR TITLE
Feature task9

### PR DESCRIPTION
**Pull request by datiasAK**
# Task 9 - React.js
## Description
One Piece website home page built using React

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
A local server using "npm start" to update the changes and React Dev Tools for Chrome and Firefox where used to test.

### CODE REVIEW

File: task09-react/src/components/ObjectSection.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        </MysticObject>
+        <MysticObject name="Gomu Gomu No Mi" img={object} alt="Gomu Gomu No Mi">
+          {description}
+        </MysticObject>
+        <MysticObject name="Gomu Gomu No Mi" img={object} alt="Gomu Gomu No Mi">
+          {description}
+        </MysticObject>
+        <MysticObject name="Gomu Gomu No Mi" img={object} alt="Gomu Gomu No Mi">
+          {description}
+        </MysticObject>
```
Comment (falvarado3) [inline task09-react/src/components/ObjectSection.js:28]: Let's try to put this into a loop so there aren't repeated lines in here, also replace the alt value with the objectName variable in the img tags,

File: task09-react/src/components/CharacterSection.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          <Character name="Portgas D. Ace" img={ace} alt="Ace" />
+          <Character name="Roronoa Zoro" img={zoro} alt="Zoro" />
+          <Character name="Nico Robin" img={nico} alt="Nico Robin" />
+        </div>
+      </article>
+    );
+  }
+}
+ 
+export default CharacterSection;
```
Comment (falvarado3) [inline task09-react/src/components/CharacterSection.js:27]: Create a const object outside the component, and define in there the 4 characters with their properties, then create a loop to create the characters in the render function.
Something like: `characters.forEach((c) => {<Character name={c.name} img={c.img} .... />})`

File: task09-react/src/components/IslandSection.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      </div>
+      <div className="article__flex article__flex--islands">
+        <Island name="Dawn Island" location="East Blue" img={dawn} alt="Dawn Island" />
+        <Island name="Reverse Mountain" location="Red Line" img={reverse} alt="Reverse Mountain" />
+        <Island name="Reverse Mountain" location="Red Line" img={reverse} alt="Reverse Mountain" />
+      </div>
+    </article>
+    );
+  }
+}
```
Comment (falvarado3) [inline task09-react/src/components/IslandSection.js:22]: The same as the character case.

File: task09-react/src/components/CharacterSection.js (hunk 2 above)
Comment (datiasAK) [reply task09-react/src/components/CharacterSection.js:27]: Thanks for the feedback. I agree. Having a loop for each character in an array makes it easy to add or remove characters without having to modify the render code.

File: task09-react/src/components/ObjectSection.js (hunk 1 above)
Comment (datiasAK) [reply task09-react/src/components/ObjectSection.js:28]: True, I forgot to use the variable I declared lines above. Also repeating those lines doesn't look good and makes it tedious to add or remove objects as they are hard coded.

**Review by falvarado3 — APPROVED** [review]

**Pull request merged**